lib/mixlib/install/generator/powershell.rb

```diff
@@ -92,7 +92,8 @@ def render_command
           <<EOS
 install -project #{options.product_name} \
 -version #{product_version} \
--channel #{options.channel}
+-channel #{options.channel} \
+-architecture #{options.architecture || 'auto'}
```

Comment:
Can we only add `-architecture #{options.architecture}` to the command when architecture exists rather than adding `"auto"` all the times? Here is what I'm thinking of:
```
install_command += "-architecture #{options.architecture}" if options.architecture